lib/prepare_maintenance_repos.pm

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Prepare maintenance updates repos.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+
+sub prepare_maintenance_repos {
```

Comment:
I was referring to put this as a new test module and add it to the schedule, we don't want libraries, they are mess, really hard to maintain in my experience in this repo (solution for that was tested a time ago like a PoC to have repo for each squad ...but never got really far) so having our own test modules is better.

Reply:
updated, thanks.